lib/contentful_rails/engine.rb

```diff
@@ -50,6 +50,15 @@ class Engine < ::Rails::Engine
       end
     end
 
+    initializer "add_contentful_mime_type" do
+      content_type = "application/vnd.contentful.management.v1+json"
+      Mime::Type.register "application/json", :contentful_json, [content_type]
```

Comment:
because we're not registering it as a synonym of :json there is no need to unregister before bulk registering? Like what we saw yesterday?